### PR TITLE
Not Checking Pending Crosslinks for First ten Epoch

### PR DESCRIPTION
**Pull request by GheisMohammadi**
## Issue
The current code attempts to delete pending crosslinks that are older than 10 epochs. However, if the pending crosslinks were not added to the database within the first 10 epochs, this can lead to an `leveldb: not found`. This PR resolves the issue by adding a condition to skip checking pending crosslinks during the first 10 epochs, ensuring we avoid unnecessary database lookups and potential errors.

### CODE REVIEW

File: node/harmony/node.go [hunk 1]
```diff
@@ -1107,8 +1107,8 @@ func New(
 	node.serviceManager = service.NewManager()
 
 	// delete old pending crosslinks
-	if node.Blockchain().ShardID() == shard.BeaconChainShardID {
-		ten := big.NewInt(10)
+	ten := big.NewInt(10)
+	if node.Blockchain().ShardID() == shard.BeaconChainShardID && node.Blockchain().CurrentBlock().Epoch().Cmp(ten) > 0 {
```
Comment (sophoah) [inline node/harmony/node.go:1111]: in localnet the crosslink hardfork epoch happens before 10th epoch, you can use `func (c *ChainConfig) IsCrossLink(epoch *big.Int) bool` to improve your code

Comment (Frozen) [reply node/harmony/node.go:1111]: image
no, it's not. What means 10 then?

Comment (sophoah) [reply node/harmony/node.go:1111]: it is [2 for localnet](https://github.com/harmony-one/harmony/blob/main/internal/params/config.go#L281), and [186 for mainnet](https://github.com/harmony-one/harmony/blob/main/internal/params/config.go#L44)

Comment (GheisMohammadi) [reply node/harmony/node.go]: done

**Review by sophoah — APPROVED** [review]

**Comment by sophoah** [comment]
@Frozen please review / approve. Thanks

File: node/harmony/node.go (hunk 1 above)
Comment (Frozen) [reply node/harmony/node.go]: 10 is magic number, you should use `IsCrossLink ` method instead.

**Review by Frozen — APPROVED** [review]

**Pull request merged**